mysql/init.sql

```diff
@@ -74,7 +74,7 @@ CREATE TABLE IF NOT EXISTS submission(
  user_id  CHAR(255)    NOT NULL,
  score CHAR(255),
  graded_date TIMESTAMP NULL ON UPDATE CURRENT_TIMESTAMP,
- local_graded_date TIMESTAMP NULL ON UPDATE CURRENT_TIMESTAMP,
+ grade_muted BOOLEAN DEFAULT TRUE,
```

Comment:
if we do not import data if the assignment is muted, we do not need this extra field.